packages/itwin/grouping-mapping-widget/src/widget/components/GroupsView.tsx

```diff
@@ -51,8 +54,11 @@ export interface GroupsViewProps {
   setSelectedGroupForDeletion: (group: Group) => void;
   onDeleteGroup: (group: Group) => Promise<void>;
   onCloseDeleteModal: () => void;
-  numberOfVisualizedGroups?: number;
+  visualizedGroups?: { numberOfVisualizedGroups: number, totalNumberOfVisualization: number };
```

Comment:
What does `totalNumberOfVisualization` mean?

Reply:
It is just the total number of groups.